crates/stackable-operator/src/builder/pod/container.rs

```diff
@@ -388,6 +436,7 @@ mod tests {
             .add_env_var_from_config_map("envFromConfigMap", "my-configmap", "my-key")
             .add_env_var_from_secret("envFromSecret", "my-secret", "my-key")
             .add_volume_mount("configmap", "/mount")
+            .unwrap()
```

Comment:
We should handle the error here.

If there is good reason not to (eg: proven not to return `Result::Err`), then we should use `expect` with a message:

```suggestion
            .expect("add volume mount")
```

Reply:
It's a test ;) Committed suggestion